nightlies/cron_wrapper

```diff
@@ -37,6 +39,15 @@ if [ $code != 0 ] ; then
 else
         # normal exit: just log stdout
         DATE=$(date)
-        echo -n "$DATE: " >> $LOG
-        cat $STDOUT  >> $LOG
+        echo -n "$DATE: "  >> $STDOUT
+        cat $STDOUT   >> $STDOUT
 fi
+
+
+# lock $LOG with file descriptor 200
+# if $LOC is locked by other process - wait for 20 sec
+exec 200>>$LOG
+flock -w 20 200 || LOG=$LOG_LOCK_ERR
+cat $STDOUT >> $LOG
+# unlock 
+flock -u 200
```

Comment:
Does this exit without error if the lock is not held (if you failed to get it above)?